components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -75,13 +73,7 @@ export class TeamManagementComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.store.dispatch(new GetTeams());
-
-    this.store.select(iamMajorVersion)
-      .pipe(takeUntil(this.isDestroyed))
-      .subscribe((majorVersion) => {
-        this.isMajorV1 = majorVersion === 'v1';
-      });
-
+    this.store.select(isIAMv2).subscribe(latest => this.isIAMv2 = latest);
```

Comment:
Placement is good. 👍 
But one more thing--just did not think to comment yesterday: you have to have the `takeUntil(this.isDestroyed)` there, otherwise you have the classic leaking subscription we have been talking about. That is, just reuse the pattern on the deleted lines L79-83.

(Applies to team-details as well.)

Reply:
oh @bcmdarroch pointed this out-- didn't realize I need both!